scripts/includes/circle-header.yml

```diff
@@ -411,15 +411,15 @@ jobs:
                     docker run --rm --privileged $BINFMT_IMAGE --install linux/amd64,linux/arm64
 
                     ## see https://docs.docker.com/desktop/multi-arch/
-                    echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+                    echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password $(aws ecr-public get-login-pw)
```

Comment:
```suggestion
                    docker login -u "$DOCKERHUB_USERNAME" --password "$DOCKERHUB_PASS"
```

Reply:
I guess I didn't technically need to change this one. Should be fine though.